dll/shellext/cabview/res/cabview.rgs

```diff
@@ -0,0 +1,47 @@
+HKCR
+{
+    NoRemove CLSID
+    {
+        '{0CD7A5C0-9F37-11CE-AE65-08002B2E1262}' = s 'Cabinet Shell Folder'
+        {
+            InprocServer32 = s '%MODULE%' { val ThreadingModel = s 'Apartment' }
+            ShellFolder
+            {
+                val Attributes = d '0x680001a0'
+            }
+            'Implemented Categories'
+            {
+                '{00021490-0000-0000-C000-000000000046}'
+                {
+                }
+            }
+        }
+    }
+
+    NoRemove CABFolder
+    {
+        CLSID = s '{0CD7A5C0-9F37-11CE-AE65-08002B2E1262}'
+        DefaultIcon = s '%MODULE%'
+
+        NoRemove Shell
+        {
+            NoRemove Open
+            {
+                val BrowserFlags = d '0x10'
+                val ExplorerFlags = d '0x20'
+                command = s '"Explorer.exe" "%%L"'
```

Comment:
Yes, /IDLIST navigates to a PIDL, the PIDL data comes from `SHAllocShared` (handle and process id). Explorer needs to call `SHLockShared` to get the data.

Reply:
Do you think we could add it here in the rgs file, even if our explorer might not yet support it?

Reply:
No, then it would not work at all.

First of all, executing `explorer.exe /idlist,:0:0,c:\` does not work in Windows nor ROS. ROS does not even parse the path bit in `SHExplorerParseCmdLine` (but it does look like it should handle the PIDL). It's our `SHELL_ArgifyW` that does %I incorrectly.

On Windows, NT5 uses DDE and NT6 uses DelegateExecute so Explorer.exe is not executed there in most cases.